Feature/sign 6807

volbobvol opened the pull request:
- Added new action parameter - "github-extended-verification-token"
- Updated documentation to cover all possible parameters

volodymyr-bobko committed:
SIGN-6808 - added github-extended-verification-token param. Updated README

volodymyr-bobko committed:
SIGN-6807 - fetching GHA token

volodymyr-bobko committed:
SIGN-6807 - updated docs

volodymyr-bobko committed:
Merge remote-tracking branch 'origin/main' into feature/SIGN-6807

paulsavoie commented on README.md:
```diff
@@ -13,14 +13,21 @@ This action allows you to sign the build artifact using SignPath signing service
-
-The [default `secrets.GITHUB_TOKEN`](https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) already includes these permissions.
+### Action parameters
+* `connector-url` (optional): The URL of the SignPath connector. Defaults to `https://api.signpath.io`
```
This is wrong, should be `https://githubactions.connectors.signpath.io`

paulsavoie committed:
minor documentation changes

paulsavoie reviewed (changes requested):
All good, changed some README and action.yml descriptions.

However, update of tests is missing

volodymyr-bobko committed:
SIGN-6807 - added unit tests for ExtendedVerificationToken

volbobvol commented:
> All good, changed some README and action.yml descriptions.
> 
> However, update of tests is missing



> All good, changed some README and action.yml descriptions.
> 
> However, update of tests is missing

Added unit tests for ExtendedVerificationToken passing through
